pkg/controllers/provisioning/binpacking/packer.go

```diff
@@ -81,7 +81,7 @@ type Packing struct {
 // to make better cost and availability decisions. The instance types returned are sorted by resources.
 // Pods provided are all schedulable in the same zone as tightly as possible.
 // It follows the First Fit Decreasing bin packing technique, reference-
-// https://en.wikipedia.org/wiki/Bin_packing_problem#First_Fit_Decreasing_(FFD)
+// https://en.wikipedia.org/wiki/First-fit-decreasing_bin_packing_(FFD)
```

Comment:
This link 404s for me.

Reply:
Works for me. Although I went here https://en.wikipedia.org/wiki/First-fit-decreasing_bin_packing

Reply:
Yup, good catch, bad copy/paste. Fixed.